tool/doc.dart

```diff
@@ -34,10 +34,15 @@ rules are considered stable, while others may be marked **experimental**
 to indicate that they are under review.
 
 Rules can be selectively enabled in the analyzer using
-[analysis options](https://pub.dartlang.org/packages/analyzer).  An
-auto-generated list enabling all options is provided
-[here](options/options.html).  As some lints may contradict each other, only
-some lints will be enabled in practice, but this list should provide a
+[analysis options](https://pub.dartlang.org/packages/analyzer)
+or through the
+[.analysis-options file](https://www.dartlang.org/guides/language/analysis-options#the-analysis-options-file). 
```

Comment:
Good catch.  That's the front-runner but there is a bit of debate still (https://github.com/dart-lang/sdk/issues/28385).  Maybe best to just ref to the dartlang doc and treat that as the source of truth (and trust that it will get updated once the dust settles around the proposed name change).  I'll follow-up with a PR.

Thanks!